opentracing-spring-web-starter/pom.xml

```diff
@@ -24,6 +43,12 @@
       <artifactId>spring-webmvc</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-webflux</artifactId>
+      <scope>compile</scope>
```

Comment:
This is default scope, so perhaps best to omit it?

Reply:
Shouldn't this be optional?

Reply:
Nice catch! Looks like I only updated the POM in `opentracing-spring-web-starter` but not in `opentracing-spring-web`. Fixed now.